lib/Kconfig.debug

```diff
@@ -356,6 +356,13 @@ config PAHOLE_HAS_BTF_TAG
 	  btf_decl_tag) or not. Currently only clang compiler implements
 	  these attributes, so make the config depend on CC_IS_CLANG.
 
+config PAHOLE_HAS_LANG_EXCLUDE
+	def_bool PAHOLE_VERSION >= 124
+	help
+	  Support for the --lang_exclude flag which makes pahole to exclude
+	  compilation units from the supplied language. Used in Kbuild to
+	  omit Rust CUs.
```

Comment:
I would take the chance to explain here why we exclude them (so that people don't need to reach for the commit message).

Then in the `pahole-flags.sh` file you could add a comment referring to that explanation, since one of the other `if`s has a comment:

```sh
	# pahole 1.18 through 1.21 can't handle zero-sized per-CPU vars
	extra_paholeopt="${extra_paholeopt} --skip_encoding_btf_vars"
```

e.g.

```sh
	# See PAHOLE_HAS_LANG_EXCLUDE
	extra_paholeopt="${extra_paholeopt} --lang_exclude=rust"
```

Or maybe the other way around (i.e. the explanation in the `.sh`), though having it in the config option may be easier for people that sees the dependency on it.

Reply:
Prefered to leave the explanation in the `Kconfig` file and refer to it in `pahole-flags.sh`, both to maintain things in one place and to explain things to `menuconfig` users.